modules/gdnative/register_types.cpp

```diff
@@ -218,9 +226,11 @@ void GDNativeExportPlugin::_export_file(const String &p_path, const String &p_ty
 				additional_code += register_pattern.replace("$name", full_name);
 			}
 			additional_code += "}\n";
-			additional_code += String("struct $prefixstruct {$prefixstruct() {add_ios_init_callback($prefixinit);}};\n").replace("$prefix", lib->get_symbol_prefix());
+			additional_code += String("struct $prefixstruct {$prefixstruct() {$callback_registration($prefixinit);}};\n").replace("$prefix", lib->get_symbol_prefix());
 			additional_code += String("$prefixstruct $prefixstruct_instance;\n").replace("$prefix", lib->get_symbol_prefix());
 
+			additional_code = additional_code.replace("$callback_registration", callback_registration_string);
+
 			add_ios_cpp_code(additional_code);
```

Comment:
shouldn't thoses `add_ios_cpp_code` & `add_ios_linker_flags` calls be behing a `if  (p_features.has("iOS"))` condition now ?

Reply:
It's a `EditorExportPlugin` method and it's also used for tvOS export to correctly register static GDNative libraries.
Maybe a more clear name should be used, like `add_ios_tvos_cpp_code` or `add_apple_platform_cpp_code`? I'm not really sure.

Reply:
yup, I think renaming would be the correct thing to do here ;-)

Reply:
On the other hand it might be better to just create `add_tvos_<cpp_code|framework|bundle_file>` methods. This way we wouldn't need to worry about breaking compatibility with already written editor plugins.

Reply:
I've added `_tvos_` methods to export plugin. This way old plugins would still work and it would also allow to use tvOS specific configuration.

Reply:
👍